### PR TITLE
Various updates on initial site content

### PR DESCRIPTION
**Pull request by vaporos**

### CODE REVIEW

**Review by peterschwarz — DISMISSED** [review]

File: community/submissions.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    generated by `git commit -s`). This sign-off is your agreement that the
+    commit satisfies the [Developer Certificate of Origin
+    (DCO)](https://developercertificate.org/).
+
+  - Your email address in the commit message must match your GitHub email
+    address. For more information, see [GitHub Help: Setting your commit email
+    address](https://help.github.com/articles/setting-your-commit-email-address-in-git/).
+
+* **Important**: If you need to rebase your pull request, be sure to rebase it
+  against the current master branch. Do not merge the current master branch
```
Comment (rbuysse) [inline community/submissions.md]: s/master/main?

File: community/planning/roadmap.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<!--
+  Copyright 2022 Cargill Incorporated
```
Comment (rbuysse) [inline community/planning/roadmap.md]: This doesn't match the range copyrights we have on the rest of the files.

File: community/licensing.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Licensing
+
+<!--
+  Copyright 2018-2022 Cargill Incorporated
```
Comment (rbuysse) [inline community/licensing.md:4]: Should this be 2021-2022 like the rest of the files?

**Review by rbuysse — CHANGES_REQUESTED** [review]

File: community/submissions.md (hunk 1 above)
Comment (vaporos) [reply community/submissions.md]: Fixed - we should update splinter-docs too.

File: community/planning/roadmap.md (hunk 2 above)
Comment (vaporos) [reply community/planning/roadmap.md]: Adjusted this.

File: community/licensing.md (hunk 3 above)
Comment (vaporos) [reply community/licensing.md:4]: The file is older than 2021, so left the copyright statement as-is.

**Review by rbuysse — APPROVED** [review]

**Review by peterschwarz — APPROVED** [review]

**Pull request merged**